src/views/Header.js

```diff
@@ -99,10 +99,22 @@ class Header extends React.Component<void, HeaderProps, void> {
     return undefined;
   }
 
+  _getHeaderTitleStyle(navigation: Navigation): ?object {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    if (header && header.titleStyle) {
+      return header.titleStyle;
+    }
+    return undefined;
+  }
+
   _renderTitleComponent = (props: SubViewProps) => {
-    const color = this._getHeaderTintColor(props.navigation);
+    let titleStyle = this._getHeaderTitleStyle(props.navigation);
```

Comment:
It seems wasteful to `getScreenConfig` for each configuration option once for each element of the screen for each screen. Especially since there is no caching.

cc @grabbou